niworkflows/anat/mni.py

```diff
@@ -107,26 +116,35 @@ def _run_interface(self, runtime):
                 self._results.update(interface_result.outputs.get())
                 NIWORKFLOWS_LOG.info(
                     'Successful spatial normalization (retry #%d).', self.retry)
+
+                if self.inputs.generate_report:
+                    try:
+                        self._generate_report()
+                    except:
+                        pass
+
```

Comment:
what is this for? Subclasses of ReportCapableInterface call self._generate_report() automatically (unless I'm missing a code path here?)